vllm/model_executor/models/mllama.py

```diff
@@ -32,8 +32,8 @@
 
 import vllm.distributed.parallel_state as ps
 from vllm.attention import Attention, AttentionMetadata, AttentionType
-from vllm.attention.backends.flash_attn import FlashAttentionMetadata
-from vllm.attention.backends.xformers import XFormersMetadata
+# from vllm.attention.backends.flash_attn import FlashAttentionMetadata
+# from vllm.attention.backends.xformers import XFormersMetadata
```

Comment:
Add conditional import
```
if current_platform.is_cuda_alike():
    from vllm.attention.backends.flash_attn import FlashAttentionMetadata
    from vllm.attention.backends.xformers import XFormersMetadata
```

Reply:
Public repo has fixed this issue since it also appears on CPU. https://github.com/vllm-project/vllm/pull/10593. We can wait for the rebase.